src/main/java/duke/tasklist/TaskList.java

```diff
@@ -0,0 +1,65 @@
+package duke.tasklist;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> arrList;
+
+    public ArrayList<Task> getList() {
+        return this.arrList;
+    }
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.arrList = tasks;
+    }
+    public TaskList() {
+        this.arrList = new ArrayList<Task>();
+    }
+    public int size() {
+        return this.arrList.size();
+    }
+
+    public Task delete(int taskNum) {
+        return this.arrList.remove(taskNum - 1);
+    }
+    public void add(Task task) {
+        this.arrList.add(task);
+    }
+
+    public void setToMark(int index) {
+        Task toMark = arrList.get(index - 1);
+        toMark.changeCompletion();
+        this.arrList.set(index-1, toMark);
+    }
+    public void setToUnmark(int index) {
+        Task toMark = arrList.get(index - 1);
```

Comment:
Maybe this can be changed into a better noun? I kind of thought that with the existence of verb methods like toString, toMark would also be considered as a verb, but I can also see that toMark can serve as a noun, so maybe keeping it as it is is good.